mlir/test/mlir-miopen-driver/sanity.mlir

```diff
@@ -14,10 +14,10 @@
 // RUN: mlir-miopen-driver -p -miopen-affix-params -miopen-lowering -miopen-affine-transform -miopen-lowering-step2 -miopen-lowering-step3 -miopen-lowering-step4 -miopen-lowering-step5 -convert-miopen-to-gpu | miopen-opt
 // RUN: mlir-miopen-driver -p -miopen-affix-params -miopen-lowering -miopen-affine-transform -miopen-lowering-step2 -miopen-lowering-step3 -miopen-lowering-step4 -miopen-lowering-step5 -convert-miopen-to-gpu -convert-gpu-to-rocdl | miopen-opt
 // RUN: mlir-miopen-driver -p -miopen-affix-params -miopen-lowering -miopen-affine-transform -miopen-lowering-step2 -miopen-lowering-step3 -miopen-lowering-step4 -miopen-lowering-step5 -convert-miopen-to-gpu -convert-gpu-to-rocdl | miopen-translate -mlir-to-rocdlir
-// RUN: mlir-miopen-driver -p -miopen-affix-params -miopen-lowering -miopen-affine-transform -miopen-lowering-step2 -miopen-lowering-step3 -miopen-lowering-step4 -miopen-lowering-step5 -convert-miopen-to-gpu -convert-gpu-to-rocdl | miopen-translate -mlir-to-rocdlir | opt -O3 -S -strip
-// RUN: mlir-miopen-driver -p -miopen-affix-params -miopen-lowering -miopen-affine-transform -miopen-lowering-step2 -miopen-lowering-step3 -miopen-lowering-step4 -miopen-lowering-step5 -convert-miopen-to-gpu -convert-gpu-to-rocdl | miopen-translate -mlir-to-rocdlir | opt -O3 -S -strip | llc -mcpu=gfx900
-// RUN: mlir-miopen-driver -p -c -target=rocdl | miopen-translate -mlir-to-rocdlir | opt -O3 -S -strip | llc -mcpu=gfx900
-// RUN: mlir-miopen-driver -p --operation conv2d -c -target=rocdl | miopen-translate -mlir-to-rocdlir | opt -O3 -S -strip | llc -mcpu=gfx900
+// RUN: mlir-miopen-driver -p -miopen-affix-params -miopen-lowering -miopen-affine-transform -miopen-lowering-step2 -miopen-lowering-step3 -miopen-lowering-step4 -miopen-lowering-step5 -convert-miopen-to-gpu -convert-gpu-to-rocdl | miopen-translate -mlir-to-rocdlir | opt -passes='default<O3>,strip' -S
+// RUN: mlir-miopen-driver -p -miopen-affix-params -miopen-lowering -miopen-affine-transform -miopen-lowering-step2 -miopen-lowering-step3 -miopen-lowering-step4 -miopen-lowering-step5 -convert-miopen-to-gpu -convert-gpu-to-rocdl | miopen-translate -mlir-to-rocdlir | opt -passes='default<O3>,strip' -S | llc -mcpu=gfx900
+// RUN: mlir-miopen-driver -p -c -target=rocdl | miopen-translate -mlir-to-rocdlir | opt -passes='default<O3>,strip' -S | llc -mcpu=gfx900
```

Comment:
It really didn't like the old separate-option syntax, I guess because -strip is considered part of -O internally, and I had to change it to the -passes option.